public/locales/en/translations.json

```diff
@@ -556,8 +556,25 @@
     },
     "flyouts": {
       "new": {
-        "title": "New Package"
+        "title": "New Package",
+        "header": "Upload a package",
+        "description": "Add a package to your solution",
+        "upload": "Upload",
+        "cancel": "Cancel",
+        "close": "Close",
+        "type": "Package type",
+        "browse": "Browse",
+        "browseText": "for a package file",
+        "placeHolder": "Select package type",
+        "package": "Package",
+        "deploymentText":"To deploy packages, go to the 'Deployments' page and click '+ New Deployment' button.",
+        "validation": {
+          "required": "Is required"
+        }
       }
+    },
+    "typeOptions": {
+      "edge_manifest": "Edge Manifest"
```

Comment:
No underscores, right?  edgeManifest?

Reply:
Currently Elvin has coded it this way in API. Once he changes, I'll also update here. Otherwise this will break the dropdown value

Reply:
We should design/write code in a way that doesn't couple the string translations with how the API has declared variables certain way. I would love to chat offline to see how we can avoid this.